Code/Framework/AzCore/AzCore/IO/Path/Path.inl

```diff
@@ -263,6 +283,11 @@ namespace AZ::IO
         return m_path.empty();
     }
 
+    constexpr const char PathView::PreferredSeparator() const noexcept
+    {
+        return m_preferred_separator;
```

Comment:
I'm guessing we went with `m_preferred_separator` instead of `m_preferredSeparator` to more closely model `std::filesystem::path::preferred_separator`?

Reply:
The original implementation was based on `std::filesystem::path` before it was transitioned over to be based on python Pathlib.